modules/react/_examples/stories/examples/MediaModal.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import {styled} from '@workday/canvas-kit-react/common';
+import {Modal} from '@workday/canvas-kit-react/modal';
+import {Box, Flex} from '@workday/canvas-kit-react/layout';
+import {Text} from '@workday/canvas-kit-react/text';
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
+import {spaceNumbers} from '@workday/canvas-kit-react/tokens';
+
+const Image = (elemProps: React.ImgHTMLAttributes<HTMLImageElement>) => {
+  const [loaded, setLoaded] = React.useState(false);
+
+  return (
+    <Box backgroundColor="frenchVanilla200">
+      <img
```

Comment:
If we use `<Box as="img"...>` can we then just use style props?

Reply:
Not sure about this one. At least, `img` element is more visual and understandable than `<Box as="img" />`, but it can look good if there is possibility to remove `Box` wrapper. 
For me below construction seems a little bit heavy
```
<Box>
  <Box as="img" {...elemProps} />
<Box>
```

compare to just `<Box as="img" {...elemProps} />`.

Reply:
yeah I'm fine with whatever makes most sense!

Reply:
I removed wrapping `Box` component and recreated `img` element as a `Box` component with style props, I also removed `alt` props because it already exists in `elemProps`